### PR TITLE
Ignore deprecated api usage from pkg:analyzer

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: source_gen/lib/src/constants/revive.dart [hunk 1]
```diff
@@ -40,6 +40,7 @@ Revivable reviveInstance(DartObject object, [LibraryElement? origin]) {
   if (element is MethodElement && element.isStatic) {
     return Revivable._(
       source: url.removeFragment(),
+      // ignore: deprecated_member_use
```
Comment (jakemac53) [inline source_gen/lib/src/constants/revive.dart:43]: can we not fix the use of the deprecated member?

Comment (kevmoo) [reply source_gen/lib/src/constants/revive.dart:43]: Sure. But we'll still have to bump the analyzer version for the next major release.

Figured it'd be better to just do the tiny thing now and roll everything later. 🤷

Comment (jakemac53) [reply source_gen/lib/src/constants/revive.dart:43]: I would just fix it now myself and up the analyzer constraint?

Comment (kevmoo) [reply source_gen/lib/src/constants/revive.dart:43]: Ugh. Fine.

**Comment by kevmoo** [comment]
@jakemac53 – PTAL

**Review by jakemac53 — APPROVED** [review]

**Pull request merged**